executor/joiner.go

```diff
@@ -203,15 +209,45 @@ func (j *semiJoiner) tryToMatch(outer chunk.Row, inners chunk.Iterator, chk *chu
 	return false, nil
 }
 
-func (j *semiJoiner) onMissMatch(outer chunk.Row, chk *chunk.Chunk) {
+func (j *semiJoiner) onMissMatchInners(outer chunk.Row, chk *chunk.Chunk) {
+}
+
+func (j *semiJoiner) tryToMatchOuters(outer chunk.Row, inners chunk.Iterator, chk *chunk.Chunk) (matched bool, err error) {
```

Comment:
Names of arguments may be `(inner, outers, ...)`, which is consistent with `joiner` interface.